src/views/MyOrganizations/MyOrganizations.vue

```diff
@@ -1,31 +1,99 @@
 <template>
     <div class="animated fadeIn">
-        <b-row>
-            <b-col>
-                <b-card>
-
-                </b-card>
-            </b-col>
-        </b-row>
+        <b-card>
+            <b-card-body>
+                <div class="btn-edit-ships d-flex justify-content-between flex-wrap mb-3">
+                    <p class="h3">My organizations</p>
+                    <div>
+                        <b-button variant="primary" role="button" @click="createOrga"><i class="fa fa-plus"></i> Create a orga</b-button>
+                    </div>
+                </div>
+                <div v-if="!listOfOrgasLoaded" class="d-flex justify-content-center">
+                    <b-spinner label="Loading..." style="width: 3rem; height: 3rem;"></b-spinner>
+                </div>
+                <div v-else>
+                    <b-row cols-lg="mt-3">
+                        <OrgaCard v-for="orga in listOfOrgas" :key="orga.id" :orga="orga"/>
+                    </b-row>
+                    <b-alert v-if="hasAnyOrga" show variant="warning">You don't have any organization yet. Why don't you create one?</b-alert>
+                    <b-alert v-if="errorMessage !== null" show variant="danger">{{ errorMessage }}</b-alert>
+                </div>
+            </b-card-body>
+        </b-card>
+        <b-modal id="modal-create-orga" ref="modalCreateOrga" size="lg" centered title="Create Orga" hide-footer>
+            <CreateOrgaModal @newOrga="onNewOrga"></CreateOrgaModal>
+        </b-modal>
     </div>
 </template>
 
 <script>
-    import placeholderShipUri from '@img/static/placeholder_ship.svg';
+    import axios from 'axios';
+    import Config from '@config/config.json';
+    import OrgaCard from '@/components/OrgaCard.vue';
+    import CreateOrgaModal from '@/components/CreateOrgaModal';
 
     export default {
-        name: 'my-organizations',
-        props: [],
-        components: {},
+        name: 'my-orga',
+        components: {CreateOrgaModal, OrgaCard},
         data() {
             return {
-                placeholderShipUri,
+                listOfOrgasLoaded: false,
+                notFoundOrgas: false,
+                errorMessage: null,
+                listOfOrgas: [],
             };
         },
         created() {
-
+            if (!this.$auth.loading) {
+                this.loadAuthRequests();
+            } else {
+                this.$auth.$on('loaded', this.loadAuthRequests);
+            }
```

Comment:
on peut supprimer tout ça (y a le auth: true dans le routing normalement)
et du coup déplacer ici le this.loadOrgaList();